deploy/katello.spec

```diff
@@ -160,19 +161,45 @@ Useful utilities for debug info collecting
 Summary: Package that installs only the Subscription and basic Content Management parts of Katello
 Group:  Applications/System
 
-# Require the common package and ensure the katello-sam package 
+# Require the common package and ensure the katello-sam package
 # can't be installed on the same system as katello
 Requires:       %{name}-common = %{version}-%{release}
 Conflicts:      %{name}
 Requires:       rubygem-hammer_cli_sam
 Requires:       %{?scl_prefix}rubygem-foreman_sam
 Requires:       sam-installer
+Requires:       %{?scl_prefix}rubygem-katello
+Requires:       rubygem-hammer_cli
+Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_katello
+Requires:       rubygem-hammer_cli_import
+Requires:       rubygem-hammer_cli_gutterball
+Requires:       %{?scl_prefix}rubygem-foreman_gutterball
 
 %description sam
 Package that installs only the Subscription and basic Content Management parts of Katello
 
 %files sam
 
+# ------ Capsule ------------------
+
+%package capsule
+Summary: Package that installs the Capsule requirements for  Katello
+Group:  Applications/System
+
+# Require the common package and ensure the katello-sam package
+# can't be installed on the same system as katello
+Requires:       %{name}-common = %{version}-%{release}
+Conflicts:      %{name}
```

Comment:
Special pre-cautions will have to be taken as we install a capsule on the main server but due to this and the requirement on 'capsule-installer' we cannot install `katello-capsule` on the main server -- https://github.com/Katello/puppet-capsule/pull/43/files#diff-a276806bea2dfcf1f57c33aee19c7309R10